app/views/tasks/_form.html.slim

```diff
@@ -113,7 +113,7 @@
                     = test.remove_nested_fields_link :text, class: 'btn btn-main btn-sm float-end' do
                       i.fa-solid.fa-xmark.text-danger
                       =< t('.button.remove_test')
-                .test-container style="margin-left: 10px; #{display_container}"
+                .test-container.ms-2.mt-4 style="#{display_container}"
```

Comment:
Aren't we aiming to get read of all inline styles? Maybe, in a follow-up PR (or another commit here), we should refactor those "dynamic" styles, too. According to my quick code search, there are still 11 of these left (at least one might be difficult regarding labels and their colors, but the remaining 10 should be refactorable, e.g., using CSS classes).

Reply:
Most of the inline styles left are triggered by javascript and/or dynamically changed. When I tried to change it to CSS classes, it just did not work as intended. 

The best way IMO is to review the changes here and if everything has been resolved, to merge it and make a new PR dedicated to modification of the js to remove remaining inline style, just for the sake of not making this PR bloated and easier to review. I will make another PR draft for the JS modification one. Is that alright?

Reply:
Sure, let's go with a follow-up PR.